framework/src/main/java/org/checkerframework/framework/type/TypeFromExpressionVisitor.java

```diff
@@ -353,10 +353,11 @@ public AnnotatedTypeMirror visitWildcard(WildcardTree node, AnnotatedTypeFactory
         // This ensures that the structure of the wildcard will match that created by
```

Comment:
The line above talks about `merge`. It also says that it does that instead of replacing them. So it is confusing that this now uses a replacer.

Reply:
I think this comment is just wrong - this PR didn't change the semantics of `AnnotatedTypeMerger`/`AnnotatedTypeReplacer` at all: the class always only did replacement.